README.md

```diff
@@ -26,8 +25,11 @@ It will also deal with hoisting the node_modules for any packages that are share
 
 1. Make sure you are using node >=12. upto 14. `node --version`.
 2. Make sure you have yarn 1.22+ installed. You do not need to install yarn 2.4+, yarn will detect the yarn 2 binary within [.yarn](./.yarn) and that will be used.
-3. Run `$ yarn` command to install all dependencies in all workspaces.
-4. Run `$ yarn build` to build all workspaces (this is needed because dependencies can depend on each other).
+3. If using the designer:
```

Comment:
Thats not going to be obvious. Could we log a similar warning when you try to start the designer? Otherwise it would be really hard for someone new to the form builder know what's happening.

Reply:
or should we invert it? So the default behavior is it works and we add a tag to say turn on CSRF?